docker/lib/dependabot/docker/file_fetcher.rb

```diff
@@ -7,7 +7,8 @@ module Dependabot
   module Docker
     class FileFetcher < Dependabot::FileFetchers::Base
       def self.required_files_in?(filenames)
-        filenames.any? { |f| f.match?(/dockerfile/i) }
+        filenames.any? { |f| f.match?(/dockerfile/i) } or
+          filenames.any? { |f| f.match?(/^[^\.]+\.ya?ml/i) }
```

Comment:
nit pick: match end of line to avoid matching something like `deployment.yml.template`?

`/^[^.]+\.ya?ml$/i`